src/domains/idpay-app/api/idpay_appio_full/openapi.appio.full.yml

```diff
@@ -1744,6 +1744,9 @@ components:
           type: string
         status:
           type: string
+          enum:
+            - AUTHORIZED
+            - REWARDED
```

Comment:
I think we should have a status for `CANCELED` transactions.

![Lista operazioni](https://github.com/pagopa/cstar-infrastructure/assets/6160324/616e1be5-244f-4259-bf2f-8b4e0c1bb30a)

Reply:
Canceled transactions are yet to be implemented
There will be a further status to be added